docs/docs/communicate/js-from-fable.md

```diff
@@ -235,6 +235,50 @@ let aDifferentName: string = import "myString" "my-lib"
 // import { myString } from "my-lib"
 ```
 
+Occasionally, you may need to import JS purely for its side-effects. For example, some browser polyfills don't export any functions, but extend the browser's built-in DOM types when executed.
+
+For this, use `importSideEffects`.
+
+```fsharp
+open Fable.Core.JsInterop
+
+importSideEffects("my-polyfill-library") // from npm package
+
+importSideEffects("./my-polyfill.js") // from local .js file
+```
+
+### Notes for Nuget Package Maintainers
+
+If you are producing a `nuget` package:
+
+- Convert any `import("package-name")` calls to `import("./package-name.js")`
+- If necessary, make sure that "./package-name.js" is copied into the project's source directory (perhaps as a pre-build step)
+- Ensure that your project file (`.fsproj`) contains these lines:
+
+```xml
+  <!-- Add source files to "fable" folder in Nuget package -->
+  <ItemGroup>
+    <Content Include="*.fsproj; **\*.fs; **\*.fsi; *.js;" PackagePath="fable\" />
+  </ItemGroup>
+```
+
+You will also find the npm package `fable-publish-utils` very useful:
+
+```shell
+$ npm install --save-dev fable-publish-utils
+```
+
+Example usage, for your `build.fsx` file:
+
+```fsharp
+open PublishUtils
+
+Target.create "publish" (fun _ ->
+    let pkg = "YourPackage"
+    pushNuget ("src" </> pkg </> pkg + ".fsproj") [] doNothing
+)
+```
```

Comment:
I don't think this is the right place to add this. 

This is a bad habit to copy a package JavaScript file into your Nuget project because this means that if there is a new version of the NPM package released you have to copy the file again inside your Nuget package and re-publish it.

If the poly-fill is available on NPM you should make it a requirement from the user to add it to the NPM package to their `package.json` file. You can use [Femto ](https://github.com/Zaid-Ajaj/Femto/)to help make this process easier. 

The note about `fable-publish-utils` would be better on this page [Author a Fable library](https://fable.io/docs/your-fable-project/author-a-fable-library.html). Although more information would need to be added to the documentation like the required ENV variables etc.